alacritty/src/renderer/solidrect.rs

```diff
@@ -0,0 +1,200 @@
+use alacritty_terminal::term::SizeInfo;
+
+use crate::gl;
+use crate::gl::types::*;
+use crate::renderer::rects::RenderRect;
+use crate::renderer::RectShaderProgram;
+
+use std::mem::size_of;
+use std::ptr;
+
+const MAX_U16_INDICES: usize = 65536;
```

Comment:
This seems like a strange constant. Wouldn't you just use `u16::max_value()` instead? Though I don't see how the maximum u16 is relevant here, this is just an arbitrary number of vertices, right?